download/tests/read-proxy-env.rs

```diff
@@ -0,0 +1,50 @@
+#![cfg(feature = "reqwest-backend")]
+
```

Comment:
separately, this is going to make a separate integration test binary; rather than a unit test. I think this is ok for now, but having it in a larger compilation unit would help prevent proliferation, but we'll need to integrate in with currentprocess() to do that - I haven't looked at env_proxy to see whats involved there.